ddtrace/tracer.py

```diff
@@ -246,6 +248,25 @@ def current_trace_context(self, *args, **kwargs):
             return active.context
         return None
 
+    def get_log_correlation_context(self):
+        # type: () -> DDLogRecord
+        """Retrieves the Correlation Identifiers for the current active ``Trace``
+        This helper method generates a DDLogRecord for custom logging instrumentation including the trace id and
+        span id of the current active span, as well as the configured service, version, and environment names.
+        If there is no active span, an empty DDLogRecord will be returned.
+        """
+        span = None
+        if self.enabled:
```

Comment:
This behaviour is a little strange IMO - mostly because the semantics of `tracer.enabled` aren't really specified. Technically the active span is still tracked and updated even if the tracer is "disabled" but since the spans won't be sent this decision makes sense :+1:

We should have a test case asserting this behaviour.

Reply:
Yup, added one: `test_get_log_correlation_context_disabled_tracer` 😄